base/windows/shop/SellRepairMenu.ts

```diff
@@ -255,7 +255,7 @@ export class SellRepairMenu {
                         this.on_character_select.bind(
                             this,
                             "sell_follow_up",
-                            this.selected_char_index,
+                            // this.selected_char_index,
```

Comment:
If not necessary, we can remove it.

Reply:
I made that commit before your Discord reply -- let me rebase.

Reply:
Done.